httprequest_lego_provider/views.py

```diff
@@ -101,3 +104,17 @@ class DomainUserPermissionViewSet(viewsets.ModelViewSet):
     queryset = DomainUserPermission.objects.all()
     serializer_class = DomainUserPermissionSerializer
     permission_classes = [IsAdminUser]
+
+
+class UserViewSet(viewsets.ModelViewSet):
+    """Views for the User.
+
+    Attributes:
+        queryset: query for the objects in the model.
+        serializer_class: class used for serialization.
+        permission_classes: list of classes to match permissions.
+    """
+
+    queryset = User.objects.all().order_by("-date_joined")
```

Comment:
Out of curiosity, may I ask what the dash in `-date_joined` is used for?
I'm not too familiar with django - from documentation I see the field [date_joiend](https://docs.djangoproject.com/en/5.0/ref/contrib/auth/#django.contrib.auth.models.User.date_joined) and was wondering if it was a typo :D

Reply:
Oh it's used for sorting in descending order on the selected field! :)